src/main/webapp/style.css

```diff
@@ -48,7 +49,7 @@ p {
 }
 
 a {
-  color: #856c8b;  
+  color: white;  
```

Comment:
If you change the color here, it will change the color for all pages that use this element, which is not what we want.